src/atoms/LinkWrapper/link_wrapper.module.scss

```diff
@@ -1,6 +1,18 @@
 @import 'assets/stylesheets/global';
 
+@mixin createColors( $type ) {
+  @for $i from 1 through 6 {
```

Comment:
@jmcolella ☝️  You should be able to dynamically iterate over these now instead of explicitly stating a number, no?